test/edu/umass/cs/gnsclient/client/singletests/AclAllFieldsSuperuser.java

```diff
@@ -79,7 +79,7 @@ public void test_100_LookupMasterGuid() {
   @Test
   public void test_142_ACLCreateAnotherGuid() {
     try {
-      String barneyName = "barney" + RandomString.randomString(6);
+      String barneyName = "barney" + RandomString.randomString(12);
```

Comment:
Minor: Good to have a single variable that controls the length of random string.